sdk/go/common/workspace/project.go

```diff
@@ -202,12 +205,14 @@ func (ps *ProjectStack) Save(path string) error {
 
 type ProjectRuntimeInfo struct {
 	name    string
+	version *semver.Version
 	options map[string]interface{}
 }
 
-func NewProjectRuntimeInfo(name string, options map[string]interface{}) ProjectRuntimeInfo {
+func NewProjectRuntimeInfo(name string, version *semver.Version, options map[string]interface{}) ProjectRuntimeInfo {
```

Comment:
Wouldn't it be better if we now created a struct that we can grow with the new options required rather than have a break of the signature each time we pass something in?

Reply:
Well at that point I think this is just the ProjectRuntimeInfo struct. 
I'm wondering if we should just leave this method as it was to not change the public API here and maybe at some point just remove it altogether. Is it a good go-ism to have a "NewX" method and an exported "X struct"?